awx/main/tasks/system.py

```diff
@@ -81,6 +81,10 @@
 def dispatch_startup():
     startup_logger = logging.getLogger('awx.main.tasks')
 
+    # TODO: Enable this on VM installs
```

Comment:
Enable this, as in... have VMs write their receptor config on startup? Implication... stop writing the receptor config for usual installs with the installer? Is that right?

Reply:
yeah exactly that, once we extend this feature to VM based installs, we'll need the ability to dynamically write receptor's conf